Koha/Plugin/EDS/bootstrap/modules/eds-results.tt

```diff
@@ -36,8 +52,8 @@
 
     <div class="main">
         <ul class="breadcrumb">
-            <li><a href="/cgi-bin/koha/opac-main.pl">[% EDSLANG.home_breadcrumb %]</a> <span class="divider">&rsaquo;</span></li>
-            <li>
+            <li class="breadcrumb-item"><a href="/cgi-bin/koha/opac-main.pl">Home</a></li>
```

Comment:
Another instance of the language template being overwritten with hardcoded language.

Reply:
Review comments fixed.